wagon-providers/wagon-http-lightweight/src/test/java/org/apache/maven/wagon/providers/http/TckTest.java

```diff
@@ -20,6 +20,7 @@
  */
 
 import org.apache.maven.wagon.tck.http.GetWagonTests;
+import org.apache.maven.wagon.tck.http.HttpsClientCertGetWagonTests;
```

Comment:
I don't see why this is necessary. Could you please explain?

Reply:
good catch, i think i had attempted to enable that test profile for the lightweight provider, but considering your explanation of when it's used, it's out of scope anyhow.